src/Layout/index.js

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import CookieConsent from 'react-cookie-consent';
```

Comment:
Everything uses the `Layout`? Is there any way to create a page w/o `Layout`?

Reply:
Everything uses layout except for the 404 page, which is just:

```
<script type = "text/javascript">
window.location = "https://developers.libra.org/blog/";
</script>
```